infra/aws/terraform/registry.k8s.io/main.tf

```diff
@@ -69,8 +66,68 @@ module "us-east-2" {
     aws = aws.us-east-2
   }
 
-  region = "us-east-2"
   prefix = var.prefix
+
+  s3_replication_iam_role_arn = "arn:aws:iam::513428760722:role/registry.k8s.io_s3admin"
```

Comment:
@deobieta, I've added sufficient permissions to the _registry.k8s.io_s3writer_ role for replication, please update this to use that role instead.

Reply:
I changed the role to "arn:aws:iam::513428760722:role/registry.k8s.io_s3writer".